lib/datarockets_style/version.rb

```diff
@@ -1,3 +1,3 @@
 module DatarocketsStyle
-  VERSION = "1.1.0".freeze
+  VERSION = "1.0.0".freeze
```

Comment:
we've already released `1.1.0` so we should not downgrade the version

Reply:
@roman-dubrovsky my bad, fixed